### PR TITLE
Added JUnit time attribute

### PR DESCRIPTION
**Pull request by spolischook**

### CODE REVIEW

File: src/Behat/Behat/Output/Node/Printer/JUnit/JUnitFeaturePrinter.php [hunk 1]
```diff
@@ -30,9 +31,15 @@
      */
     private $statistics;
 
-    public function __construct(PhaseStatistics $statistics)
+    /**
+     * @var JUnitDurationListener
+     */
+    private $durationListener;
+
+    public function __construct(PhaseStatistics $statistics, JUnitDurationListener $durationListener)
```
Comment (everzet) [inline src/Behat/Behat/Output/Node/Printer/JUnit/JUnitFeaturePrinter.php]: second argument must be optional for BC

File: src/Behat/Behat/Output/Node/Printer/JUnit/JUnitScenarioPrinter.php [hunk 2]
```diff
@@ -47,10 +48,16 @@
      */
     private $outlineStepCount;
 
-    public function __construct(ResultToStringConverter $resultConverter, JUnitOutlineStoreListener $outlineListener)
+    /**
+     * @var JUnitDurationListener
+     */
+    private $durationListener;
+
+    public function __construct(ResultToStringConverter $resultConverter, JUnitOutlineStoreListener $outlineListener, JUnitDurationListener $durationListener)
```
Comment (everzet) [inline src/Behat/Behat/Output/Node/Printer/JUnit/JUnitScenarioPrinter.php]: Third argument must be optional for BC

File: src/Behat/Behat/Output/Node/Printer/JUnit/JUnitFeaturePrinter.php (hunk 1 above)
Comment (spolischook) [reply src/Behat/Behat/Output/Node/Printer/JUnit/JUnitFeaturePrinter.php]: Fixed

File: src/Behat/Behat/Output/Node/Printer/JUnit/JUnitScenarioPrinter.php (hunk 2 above)
Comment (spolischook) [reply src/Behat/Behat/Output/Node/Printer/JUnit/JUnitScenarioPrinter.php]: Fixed

**Comment by christeredvartsen** [comment]
Any status update for this?

**Comment by ciaranmcnulty** [comment]
Thank you @spolischook, sorry it took so long to merge

**Pull request merged**